packages/CalendarCarousel/index.tsx

```diff
@@ -202,6 +226,35 @@ function CalendarCarousel<T>({
       numPrevMonthDays--;
     }
 
+    const markedDayEvents = [
+      {
+        selectedEventDate: new Date(2020, 7, 7),
+        events: 'walk dog',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 17),
+        events: 'birthday',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 27),
+        events: 'cooking',
+      },
+    ];
+
+    const [eventSwitch, setEventSwitch] = useState(0);
+    const dayEvents = [];
+    const markeddates = markedDayEvents.map((markeddates) => markeddates.selectedEventDate.getDate());
+    const markedmonths = markedDayEvents.map((markedmonths) => markedmonths.selectedEventDate.getMonth() - 1);
+    const isInside = (d: number) : boolean => {
+      if (markeddates.includes(d) && markedmonths.includes(month)) { return true; }
+      return false;
+    };
+    for (let i = 0; i <= markedDayEvents.length - 1; i++) {
+      if (markeddates[i] === eventSwitch) {
+        const getEventDay = markedDayEvents[i].events; dayEvents.push(getEventDay);
```

Comment:
New line after `events;`

Reply:
https://github.com/dooboolab/dooboo-ui/pull/269#discussion_r462827520

Reply:
thank you for your help fixedthis!